React/recap-projekt-4/color-theme-projekt/src/ThemeCard/ThemeCard.js

```diff
@@ -0,0 +1,12 @@
+import ColorCard from "../ColorCard/ColorCard";
+
+export default function ThemeCard({ theme }) {
+  return (
+    <>
+      <h2>{theme.name}</h2>
+      {theme.colors.map((color) => (
+        <ColorCard key={color.name} color={color} />
```

Comment:
the color name might not be unique depeding on the colors in the theme.
```suggestion
        <ColorCard key={color.role} color={color} />
```